src/main/resources/network/brightspots/rcv/GuiConfigLayout.fxml

```diff
@@ -539,7 +539,7 @@
             <VBox>
               <VBox maxWidth="460.0" styleClass="bordered-box">
                 <HBox spacing="4.0">
-                  <Label prefWidth="212.0" text="Overvote Rule *"/>
+                  <Label prefWidth="100.0" text="Overvote Rule *"/>
```

Comment:
I'll need to check this in Linux tomorrow before approving.

Reply:
Yeah, this cuts things off in Linux. Please revert to what it was before.

Reply:
120.0 also works ok, which is closer, but isn't in line with the other input box. Either that or 212 is ok with me.

Reply:
OK, going with 120.